config/initializers/devise.rb

```diff
@@ -260,3 +260,11 @@
   # so you need to do it manually. For the users scope, it would be:
   # config.omniauth_path_prefix = '/my_engine/users/auth'
 end
+
+# Hack to prevent some devise goofiness around having both activerecord
+# and mongoid. Removable when we move user class to pg.
+Devise.module_eval do 
```

Comment:
This forces a setting to avoid some truly bizarre pg-specific devise behavior - seems like a couple of the modules have completely different logic when using activerecord and not.